### PR TITLE
Renew ZooKeeper example configuration in docs

### PR DESCRIPTION
**Pull request by alexey-milovidov**
Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/operations/tips.md [hunk 1]
```diff
@@ -213,27 +213,17 @@ GROUP=zookeeper
 PIDDIR=/var/run/$NAME
 PIDFILE=$PIDDIR/$NAME.pid
 SCRIPTNAME=/etc/init.d/$NAME
-JAVA=/usr/bin/java
+JAVA=/usr/local/jdk-11/bin/java
```
Comment (kitaisreal) [inline docs/en/operations/tips.md:216]: At the top other version of jre is used.
```
Java(TM) SE Runtime Environment (build 1.8.0_25-b17)
Java HotSpot(TM) 64-Bit Server VM (build 25.25-b02, mixed mode)
```

Comment (alexey-milovidov) [reply docs/en/operations/tips.md:217]: Fixed.

**Comment by alexey-milovidov** [comment]
For some reason, our generated HTML documentation goes through Jinja. Looks like a mistake.

**Comment by gyuton** [comment]
Internal documentation ticket: DOCSUP-8448.

**Pull request merged**